### ISSUE TITLE
[Adaptive Cards- Home- Flight Itinerary]: Talkback focus move to hidden element and announce it as "departs from" using swipe right gesture.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/Teams-AdaptiveCards-Mobile#12]
### Target Platforms

Android

### SDK Version

2023.01.20.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment**
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.01.20.1

**Repro Steps:**
1. Turn on Talkback.
2.  Open Adaptive Cards Visualizer Application.
3.  Home Page will opened.
4.  Now Navigate to 'Sample card' button and select it.
5.  Now navigate to "choose sample" expands/collapse control and select "Flight Itinerary" list item.
6.  Now Navigate to Visual tab and select it.
7.  Now navigate to Flight Itinerary card using swipe gesture and observe the issue.

**Actual Result:**
Talkback focus move to hidden element using swipe right gesture. talkback focus move to hidden element after "**Tue, May 20, 2017 12:25PM"** text and announce it like **"departs from"**

Expected Result:
Talkback focus shouldn't move to hidden element using swipe right gesture. Talkback focus shouldn't move to hidden element after "**Tue, May 20, 2017 12:25PM"** text or Visually text should be appears on the card.

User Experience:
When focus order does not follow the visible reading order, the experience can create a different interaction pattern, and/or unintended content relationships for assistive technology users. On rare occasions, there is a logical reason the focus order is different from the reading order, but the meaning of the content and interactions must remain the same.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/89897257/230121731-f06e5031-a35e-400b-ad63-d9c4a70904b1.mp4

### Card JSON

```JSON
NA
```

**Comment by VikrantSingh01** [comment]
@MengliElmendorf137 : Can you pls review this.

**Comment by ardlank** [comment]
```json
{
  "type": "TextBlock",
  "text": "Departs From",
  "wrap": true,
  "style": "default",
  "weight": "bolder",
  "color": "light"
}
```

This is working as expected. There is a clash between the text color and the background color. Both are light colors. This experience exists in web, android and iOS. See screenshots below of light and dark team for Android and web.

<img src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/23054357/24914c37-3cf4-4565-8b61-9a852b5ccce7" width="300" height="620">
<img src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/23054357/61861d35-9ae7-44f0-a27f-cb68768cdca1" width="300" height="620">
<img width="1725" alt="light_web" src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/23054357/b7ee973d-8476-43f1-91c0-0f2619c9f892">
<img width="1728" alt="dark_web" src="https://github.com/microsoft/AdaptiveCards-Mobile/assets/23054357/8c8d8728-922d-4b86-ba15-fab962cb5276">

**Comment by vagpt** [comment]
Hi @ardlank,

Downgrading its severity from Sev 2 to Sev 3 as discussed with Team.

Also, this issue needs to be fixed as this will impact visually the user as well. Departs from and Arrives At label should be visible in Light mode as well as it is getting visible in Dark mode. Also, as a screen reader user they are getting infomration about the label, but it does not visually appear for them it will treat as hidden element as well. hence, reactivating this issue.


![image](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/e9e0e7bd-966e-4364-b230-ae0b04baf382)
